tabs/src/services/lnbitsServiceLocal.ts

```diff
@@ -500,6 +500,32 @@ const getWalletIdByUserId = async (adminKey: string, userId: string) => {
   }
 };
 
+const getRewards = async (apiKey: string) => {
```

Comment:
Should we change `apiKey` to either `adminKey`? Not sure what rewards uses as the X-Api-Key.